components/interactive_dialog/dialog_element/dialog_element.tsx

```diff
@@ -99,7 +114,7 @@ export default class DialogElement extends React.PureComponent {
 
         let {type, maxLength} = this.props;
 
-        let displayNameContent = displayName;
+        let displayNameContent: React.ReactElement;
```

Comment:
Anything that we're rendering can be `React.ReactNode` to keep everything consistent when we don't care about what the specific type of the variable is.
```suggestion
        let displayNameContent: React.ReactElement;
```